common/prove/witness_helper.go

```diff
@@ -351,11 +355,11 @@ func (w *WitnessHelper) constructAccountWitness(
 func (w *WitnessHelper) constructNftWitness(
 	proverNftInfo *NftWitnessInfo,
 ) (
-	// nft root before
+// nft root before
```

Comment:
The comment should be aligned with the code.

Reply:
updated